rinse/core.py

```diff
@@ -0,0 +1,203 @@
+from cookiecutter.main import cookiecutter
+from pathlib import Path
+from os import listdir, chdir, mkdir, symlink, remove, environ
+from shutil import rmtree
+from pkg_resources import resource_filename
+from rinse import cookies
+import requests as re
+import tarfile
+import subprocess as sp
+
+
+class BaseInstallR(object):
+
+    def __init__(self, path, name, version=None, repos=None, method=None, init=None):
+        # Rinse path setup
+        self.name = name
+        self.path = Path(path).expanduser().absolute()
+
+        self.rinse_path = self.path / self.name
+        self.tmp_path = self.rinse_path / "tmp"
+        self.src_path = self.rinse_path / "src"
+        self.lib_path = self.rinse_path / "lib"
+        self.bin_path = self.rinse_path / "bin"
+
+        # Initialization step
+        self.cookie_jar = Path(resource_filename(cookies.__name__, ''))
+        if init is True:
```

Comment:
```suggestion
        if init:
```
Can shorten this and make it more pythonic.